testing/nqp/APKBUILD

```diff
@@ -0,0 +1,50 @@
+# Contributor: Curt Tilmes <Curt.Tilmes@nasa.gov>
+# Maintainer: Curt Tilmes <Curt.Tilmes@nasa.gov>
+pkgname=nqp
+pkgver=2018.02
+pkgrel=0
+pkgdesc="Not Quite Perl"
+url="https://github.com/perl6/nqp"
+arch="all"
+options="!archcheck" # Arch dependencies are embedded
+license="Artistic-2.0"
+depends="moarvm"
```

Comment:
If the package uses a shared library from `moarvm`, declaring this is unnecessary.  `abuild` will automatically declare all shared library dependencies.